notebooks/text_models/solutions/keras_for_text_classification.ipynb

```diff
@@ -176,7 +176,7 @@
    "cell_type": "markdown",
```

Comment:
<p>It seems we are not using this saved data?</p><p>Maybe we can just remove this part?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>removed this! </p>